lib/src/Base/Stat/FieldImplementation.cxx

```diff
@@ -157,7 +157,7 @@ Scalar & FieldImplementation::operator () (const UnsignedInteger i,
   // No copyOnWrite() as the at() method already do it
   return at(i, j);
 #else
-  return (*this)[i][j];
+  return (*this)(i, j);
```

Comment:
@dbarbier @regislebrun I introduced an infinite recursion in Field::operator(i,j), I'll fix it quickly